src/gui/mzroll/tabledockwidget.cpp

```diff
@@ -2533,6 +2623,10 @@ void BookmarkTableDockWidget::showMsgBox(bool check, int tableNo) {
   msgBox->open();
 }
 
+
+
+
+
```

Comment:
@MdAzmatAli  Too many blank lines.. Could you please remove them? Ideally, we have just one blank line between the functions.

Reply:
@sakshikukreja14 Fixed